D5.py

```diff
@@ -0,0 +1,177 @@
+import numpy as np
+import random
+import tqdm
+from typing import List
+import scipy.stats as stats
+from validator import Validator
+from lm_proposer import GPT3_Proposer
+from sample_lexical_diversity import lexical_diversity
+from scipy.stats import norm
+
+
+def calculate_diff_w_significance(A_scores, B_scores, alpha=1e-5):
```

Comment:
Nitpick:
A and B scores should probably be lower cased https://peps.python.org/pep-0008/#function-and-variable-names